src/Credentials/CredentialProvider.php

```diff
@@ -358,6 +434,9 @@ private static function remoteCredentialProviders(array $config = [])
         if (!empty(getenv(EcsCredentialProvider::ENV_URI))) {
             $providers['ecs'] = self::ecsCredentials($config);
         }
+        $providers['process_credentials'] = self::process();
+        $providers['process_config']
+            = self::process('profile default', self::getHomeDir() . '/.aws/config');
```

Comment:
Should there be a test that checks that the process credentials are hit (both from `process_credentials` and from `process_config` on a call to `defaultProvider`?

Reply:
Yes, added.